libs/src/services/discoveryProvider/index.js

```diff
@@ -14,7 +14,6 @@ let urlJoin = require('proper-url-join')
 const DiscoveryProviderSelection = require('./DiscoveryProviderSelection')
 if (urlJoin && urlJoin.default) urlJoin = urlJoin.default
 
-const MAKE_REQUEST_RETRY_COUNT = 3
 const MAX_MAKE_REQUEST_RETRY_COUNT = 20
```

Comment:
let's drop the retries down to like 5 or something. we should probably try switching over sooner than 20 requests